api/src/main/java/jakarta/data/repository/Slice.java

```diff
@@ -48,14 +50,14 @@
     /**
      * Returns the current {@link Pageable}
      *
-     * @return the current Pageable
+     * @return the current Pageable; will never be {@literal null}.
      */
     Pageable getPageable();
 
     /**
      * Returns the next {@link Pageable#next()}, or <code>null</code> if it is known that there is no next page.
      *
-     * @return the next pageable
+     * @return the next pageable; will never be {@literal null}.
```

Comment:
This is incorrect and contradicts two lines up where it explicitly states,
"or <code>null</code> if it is known that there is no next page."